spyder/plugins/layout/container.py

```diff
@@ -20,6 +20,7 @@
 from spyder.api.exceptions import SpyderAPIError
 from spyder.api.translations import _
 from spyder.api.widgets.main_container import PluginMainContainer
+from spyder.config.utils import (is_wsl)
```

Comment:
```suggestion
from spyder.config.utils import is_wsl
```

No need of parentheses here.